First-outlook.md

```diff
@@ -46,18 +45,16 @@ Sidenote (Windows): When using HashLink you can run them very quickly when [addi
 ### 🌐 Community
 
 - Ask questions, discuss and get in touch with the **community**!
-  - https://community.heaps.io/
-  - join our **discord** channel: https://community.heaps.io/t/discord-official-channel/72
-    - we also have Haxe game jams in the "JAM TIME" channel!
+  - Discord (*#heaps* channel): https://discord.gg/sWCGm33
```

Comment:
The Discord link is the one that is given in the discourse thread -  https://community.heaps.io/t/discord-official-channel/72